tromega/lib/data/tracking_http_helper.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:tromega/data/execution.dart';
+import 'package:tromega/data/trainingSession.dart';
+import 'package:http/http.dart' as http;
+import 'dart:convert';
+
+class TrackingHttpHelper {
+  //final String authority = 'api.fitnessapp.gang-of-fork.de';
+  final String authority = 'virtserver.swaggerhub.com';
+  final String path = '/FLORIANHASE12/GEtit/1.0.0';
```

Comment:
could be stored in .env 
[HowTo](https://stackoverflow.com/questions/60800930/how-do-i-define-a-variable-inside-an-env-file-in-flutter)

Reply:
@lumaghg was meinst du zur Auslagerung in die .env?

Die condition arbeite ich noch ein und commite dann. Danke @steffenhls

Reply:
@MatthiasBiermanns @steffenhls 
Wofür sind diese authority und path Variablen da?

Reply:
@lumaghg die entsprechen nur der URL der abzufragenden API

Reply:
@lumaghg @steffenhls 
Habe es schon fast vermutet, aber danke für die Bestätigung. Sofern @STAT1C-Sy nichts dagegen hat, würde ich es dann auch so lassen

Reply:
Gebe @lumaghg und @MatthiasBiermanns da Recht man könnte nur überlegen alles was Config related ist in ein config file/Config Klasse auszulagern dann hat mans zentral

Reply:
@projekt-fitnessapp/leitung das hier ist übrigens ein traumhaftes Beispiel für die Kommunikation mit Hilfe von PRs :-)

Reply:
Finde die Idee von @STAT1C-Sy  auch gut, denke aber fürs erste ist das unnötig kompliziert. Habe gerade mal geguckt wie es gehen würde und das bringt ggf. weitere Probleme mit sich. Lasse es jetzt erstmal so, können in Zukunft (maybe 6. Semester gerne nochmal n Blick drauf werfen)

Reply:
@MatthiasBiermanns klingt gut nehmen wir mit in die nächste Woche ;)